lib/validator.js

```diff
@@ -82,6 +84,7 @@ function executeValidator(ctx, name, attribute, setValue) {
     } catch (err) {
       // revert value (instance only)
       if (ctx.raw && needToRevert) ctx.raw[field] = originValue;
+      err.name = ERR_NAME;
```

Comment:
`error.name` should be identifiable, thus developer can use it for properly response.